README.md

```diff
@@ -6,39 +6,38 @@ Everything we know about frontend, backend, UX, and Devops consulting and manage
 
 The following sections detail how to make various improvements to the site. Make sure you have cloned this repo and run:
 
-```
+```sh
 npm install
 ```
 
-### Watch mode
+### Dev Server
 
-When actively working on the content, the most useful option is the watch mode. Run:
+You can start a development server at `http://localhost:8080/academy/` by running:
 
-```
-npm run dev
+```sh
+npm run start
 ```
 
-This will take a while the first time. The site will be created in the `academy` folder.
-
-If the upcoming calendar events section returns "Sorry, events can’t load right now", it may be an issue with the ip address, so try http://localhost:5500/academy/
+`start` incorporates the `npm run dev` command and changes to the source code will cause a rebuild,
+however the browser will **not** reload when the source code changes.
 
-### Dev Server
+### Watch mode
 
-You can start a development server at `http://localhost:8080/` by running:
+When actively working on the content, the most useful option is the watch mode. Run:
```

Comment:
> the most useful option is the watch mode

Should this be updated to explain the difference with `start` above?

Reply:
Updated in the next PR.